Python/Product/Debugger/Debugger/PythonThread.cs

```diff
@@ -31,34 +33,30 @@ class PythonThread {
             _name = "";
         }
 
-        public void StepInto() {
-            _process.SendStepInto(_identity);
+        public Task StepIntoAsync(CancellationToken ct) {
+            return _process.SendStepIntoAsync(_identity, ct);
```

Comment:
The => syntax also works for methods (as well as read-only properties), if you wanted to simplify these down. I don't have any preference either way (though I do prefer not using async and await for trivial functions like these)